### PR TITLE
spider: store desk for http response

### PR DESCRIPTION
**Pull request by liam-fitzgerald**
Fixes spider to scry the mark conversion from the correct desk

### CODE REVIEW

File: pkg/arvo/app/spider.hoon [hunk 1]
```diff
@@ -193,6 +204,23 @@
...
+      ?:  ?=(%3 -.old)
+        `old
+      :-  ~
+      :*  %3
+        starting.old
+        running.old
+        tid.old
+        %-  ~(gas by *(map tid [@ta =mark =desk]))
+        %+  turn  ~(tap by serving.old)
+        |=([=tid id=@ta =mark] [tid id mark q.byk.bowl])
```
Comment (belisarius222) [inline pkg/arvo/app/spider.hoon:221]: We should remember to migrate HTTP bindings for threads to their new desks during the dist OTA.

**Review by belisarius222 — APPROVED** [review]
LGTM

**Review by Fang- — APPROVED** [review]
This checks out.

File: pkg/arvo/app/spider.hoon (hunk 1 above)
Comment (liam-fitzgerald) [reply pkg/arvo/app/spider.hoon:221]: We don't necessarily need to do this, as all threads called via HTTP resolve in a single event.

**Pull request merged**